tests/phpunit/tests/query.php

```diff
@@ -897,4 +897,70 @@ public function test_query_tag_404_does_not_throw_warning() {
 		$this->assertFalse( $q->is_tax() );
 		$this->assertFalse( $q->is_tag( 'non-existent-tag' ) );
 	}
+
+	/**
+	 * Test if $before_loop is true before loop.
+	 *
+	 * @ticket 58211
+	 *
+	 * @covers WP_Query::before_loop
```

Comment:
```suggestion
```
`@covers ClassName::<public>` is not recommended as per [document](https://docs.phpunit.de/en/9.5/annotations.html#covers) and WP did not use it anywhere.

Reply:
Covers are designed for functions and methods. Not properties of a class. See https://docs.phpunit.de/en/9.5/annotations.html#covers